app/client/components/ActionLog.ts

```diff
@@ -2,23 +2,25 @@
  * ActionLog manages the list of actions from server and displays them in the side bar.
  */
 
+import {GristDoc} from 'app/client/components/GristDoc';
 import * as dispose from 'app/client/lib/dispose';
-import dom from 'app/client/lib/dom';
-import {timeFormat} from 'app/common/timeFormat';
-import * as ko from 'knockout';
-
 import koArray from 'app/client/lib/koArray';
 import {KoArray} from 'app/client/lib/koArray';
 import * as koDom from 'app/client/lib/koDom';
-
-import {GristDoc} from 'app/client/components/GristDoc';
+import {makeT} from 'app/client/lib/localization';
+import {ClientTimeData} from 'app/client/models/TimeQuery';
+import {basicButton} from 'app/client/ui2018/buttons';
+import {labeledSquareCheckbox} from 'app/client/ui2018/checkbox';
 import {ActionGroup} from 'app/common/ActionGroup';
 import {ActionSummary, asTabularDiffs, defunctTableName, getAffectedTables,
         LabelDelta} from 'app/common/ActionSummary';
 import {CellDelta, TabularDiff} from 'app/common/TabularDiff';
-import {DomContents, fromKo, IDomComponent} from 'grainjs';
-import {makeT} from 'app/client/lib/localization';
-import {labeledSquareCheckbox} from 'app/client/ui2018/checkbox';
+import {timeFormat} from 'app/common/timeFormat';
+import { ResultRow, TimeCursor, TimeQuery} from 'app/common/TimeQuery';
+import {dom, DomContents, fromKo, IDomComponent, makeTestId, styled} from 'grainjs';
+import * as ko from 'knockout';
+
+const testId = makeTestId('ActionLog_');
```

Comment:
```suggestion
const testId = makeTestId('test-actionlog-');
```

Also, I don't see it being exercised anywhere, so maybe we can remove it.